packages/core/test/specs/data_sources/model/StyleDataVariable.ts

```diff
@@ -183,7 +183,7 @@ describe('StyleDataVariable', () => {
         content: 'Hello World',
       })[0];
 
-      em.getEditor().CssComposer.addCollection([
+      const [rule] = em.getEditor().CssComposer.addCollection([
```

Comment:
@artf this pr is related to our discussion about getting CssRuleView to listen to a DataSource. 

Is it a 1-1 mapping from CssRule to CssRuleView? 

What assertions could we add in this file against the view? 

We already check pre and post update that the CSS is correct plus the rule.style is updated.

Reply:
I can compose a: 

```ts
const cssRuleView = new CssRuleView({ model: rule, config: rule.config });
```

From the rule, but it looks like there is no direct correlation to the view from the rule. 

In other DataSource test files we assert the component and the view like this for example: 

```ts
const style = cmp.getStyle();
expect(style).toHaveProperty('color', 'red'); // assuming this is the model
expect(cmp.getEl()?.innerHTML).toContain('Hello World'); // assuming this is the view
```

What other assertions could I make other than the: 

```ts
expect(rule.getStyle()).toHaveProperty('color', 'red');
expect(em.getEditor().getCss()).toContain(`${selector}{color:red;}`); // this is the view?
```

For a CSS Rule?

Reply:
It looks here that the CssRuleView is created once, rendered, and then forgot about 

https://github.com/GrapesJS/grapesjs/blob/336f4b81e66206dcd3e067c8f94bf2d0e89533f3/packages/core/src/css_composer/view/CssRulesView.ts#L77-L84

I was wondering how we intended to attach watchers to these.

Reply:
Yeah, correct, we don't have any relation between CSSRule and CSSRuleView.
I think we can establish a similar relationship with Component and ComponentView (we're adding the view [here](https://github.com/GrapesJS/grapesjs/blob/025fd7dcb5dd47646ffe419b674f6e964dd1902f/packages/core/src/dom_components/view/ComponentView.ts#L81) and removing it [here](https://github.com/GrapesJS/grapesjs/blob/025fd7dcb5dd47646ffe419b674f6e964dd1902f/packages/core/src/dom_components/view/ComponentView.ts#L155)). 
It's a 1-to-many mapping (model could have multiple views) and once we have it we could probably trigger the dataSource update directly from the model side (eg. `model.views.forEach(view => view.renderStyles())`). wdyt?